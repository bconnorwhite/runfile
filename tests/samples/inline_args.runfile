# ----------
# Inline Args and Flags
# ----------

# This is about the command
command arg -f, --flag -o, --other:
  echo "Running command with inline args and flags"
  echo "arg: $arg"
  echo "flag: $flag"
  echo "other: $other"

# Another command with inline args
test input? -v, --verbose --output=<file>:
  echo "Testing with inline args"
  echo "input: $input"
  echo "verbose: $verbose"
  echo "output: $output"
