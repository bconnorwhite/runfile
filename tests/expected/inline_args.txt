Inline Args and Flags
  command           # This is about the command
    arg
    -f, --flag
    -o, --other
  test              # Another command with inline args
    input?
    -v, --verbose
    --output

